lib/graphql/introspection/type_kind_enum.rb

```diff
@@ -1,7 +1,7 @@
 GraphQL::Introspection::TypeKindEnum = GraphQL::EnumType.define do
   name "__TypeKind"
-  description "The kinds of types in this GraphQL system"
+  description "An enum describing what kind of type a given `__Type` is."
   GraphQL::TypeKinds::KIND_NAMES.each do |kind_name|
-    value(kind_name)
+    value(kind_name, GraphQL::TypeKinds::TYPE_KIND_DESCRIPTIONS[kind_name.to_sym])
```

Comment:
It looks like `KIND_NAMES` is used here and in `TypeKind` for dynamically generating some methods. 

I think it would be a nice refactor to: 

- Add `TypeKind#description` (bringing it inline with other schema members) and initialize each one with its description
- Remove `KIND_NAMES` and iterate over `TYPE_KINDS` here & in `TypeKind` instead 

Or, is there a benefit to keeping the descriptions in a hash?